api/api-rules/violation_exceptions.list

```diff
@@ -640,6 +640,7 @@ API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,V
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,VolumeConfiguration,FlexVolumePluginDir
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,VolumeConfiguration,PersistentVolumeRecyclerConfiguration
 API rule violation: names_match,k8s.io/kube-proxy/config/v1alpha1,KubeProxyConfiguration,IPTables
+API rule violation: names_match,k8s.io/kube-scheduler/config/v1alpha1,KubeSchedulerConfiguration,PodMaxBackoffDurationSeconds
```

Comment:
Can we avoid violating the API rules?

Reply:
I think we can't.  As @alculquicondor mentioned in https://github.com/kubernetes/kubernetes/pull/81263#discussion_r316189923,  just putting json name to `PodMacBackoffDurationSeconds` field can resolve this.

Reply:
done